### ISSUE TITLE
Example send directory with tar + curl

### ISSUE THREAD

**Issue by abitrolly** [dutchcoders/transfer.sh#230]
```
# Archive and upload directory
tar -czf - /var/log/journal | curl --upload-file - https://transfer.sh/journal.tar.gz
```

**Comment by dopessoa** [comment]
@aspacca,

How about creating an examples.md file and referencing it on the README.md? 

The idea would be to leave the main examples on the README.md and put the other ones on the examples.md.

**Comment by aspacca** [comment]
@dopessoa yes, good idea. feel free to open a PR for that: I don't know how much time I will have for doing it promptly at the moment

**Comment by dopessoa** [comment]
I was really busy with work, I just could get back to this now, sent a PR, please review it when you have the time.

**Comment by aspacca** [comment]
fixed in #259 . thanks @dopessoa
